snuba/cli/cleanup.py

```diff
@@ -19,6 +19,18 @@
     type=int,
     help="Clickhouse native port to write to.",
 )
+@click.option(
+    "--clickhouse-user",
+    default=settings.CLICKHOUSE_USER,
+    type=str,
+    help="Clickhouse username.",
+)
+@click.option(
+    "--clickhouse-pass",
+    default=settings.CLICKHOUSE_PASS,
+    type=str,
+    help="Clickhouse password.",
+)
```

Comment:
Please let's not add user name and password to the CLI (for any command). This has two drawbacks:
1) security. Adding passwords in command lines leaks them to whoever has access to the list of running processes on a system and to the shell history. Using settings and environment variables are better practices.
2) Snuba configuration plan. We are actually removing Clickhosue connection settings from the command line everywhere (see some progress here: https://github.com/getsentry/snuba/pull/897).